src/templates/plugin.jsx

```diff
@@ -95,12 +99,13 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
                                 installations={plugin.stats.installations}
                             />
                         </div>
+                        <div className="label-link"><a target="_blank" href={`https://stats.jenkins.io/pluginversions/${plugin.name}.html`} rel="noreferrer">Show details</a></div>
                     </div>
                     <div className="sidebarSection">
                         <h5>Links</h5>
-                        {plugin.scm && plugin.scm.link && <div className="label-link"><a href={plugin.scm.link}>GitHub</a></div>}
+                        {plugin.scm && plugin.scm.link && <div className="label-link"><a target="_blank" href={plugin.scm.link} rel="noreferrer">GitHub</a></div>}
```

Comment:
I'm generally not a fan of forcing new tabs to happen, takes away the agency from the user. Is there a request for new tab, or just a preference?